content/v2.0/query-data/guides/monitor-states.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Monitor states
+seotitle: Monitor states and state changes in your events and metrics with Flux.
+description: Flux provides several functions to help monitor states and state changes in your data.
+v2.0/tags: [states, monitor, flux]
+menu:
+  v2_0:
+    name: Monitor states
+    parent: How-to guides
+weight: 209
+---
+
+Flux helps you monitor states in your metrics and events:
+
+- [Find how long a state persists](#find-how-long-a-state-persists)
+- [Count the number of states](#count-the-number-of-states)
+- (Cloud only) [Detect state changes](#detect-state-changes)
+
+If you're just getting started with Flux queries, check out the following:
+
+- [Get started with Flux](/v2.0/query-data/get-started/) for a conceptual overview of Flux and parts of a Flux query.
+- [Execute queries](/v2.0/query-data/execute-queries/) to discover a variety of ways to run your queries.
+
+## Find how long a state persists
+
+1. In your query, specify the bucket to search.
+2. Specify a time range to search.
+3. Use the `stateDuration()` function and include the following information:
+  
+  - **Column to search:** any tag key, tag value, field key, field value, or measurement.
+  - **Value:** value (or state) to search for in the specified column.
+  - **State duration column:** a new column to store the state duration─the length of time the specified value persists.
+  - **Unit:** of time to measure the state duration (`1s` (by default), `1m`, `1h`).
+
+        ```bash
+        |> stateDuration(fn: (r) => r._column_to_search == "value_to_search_for", column: "state_duration", unit: 1s
+        ```
+
+4. Run `stateDuration()` to search each point in the specified time range for the specified value:
+
+    - For the first point that evaluates `true`, the state duration is set to `0`. For each consecutive point that evaluates `true`, the state duration increases by the time interval between each consecutive point (in specified units).
+    - If the state is `false`, the state duration is reset to `-1`.
+  
+{{% truncate %}}
```

Comment:
Not sure any `truncate` blocks are needed in this doc.

Reply:
I'll remove the truncate blocks....my goal was to minimize scrolling in preparation for more examples to come. One to discuss for future.